src/main/java/org/folio/circulation/domain/notice/TemplateContextUtil.java

```diff
@@ -108,6 +109,30 @@ public static JsonObject createStaffSlipContext(Request request) {
     return createStaffSlipContext(request.getItem(), request);
   }
 
+  public static JsonObject addPrimaryServicePointNameToStaffSlipContext(JsonObject entries, ServicePoint primaryServicePoint) {
+    log.info("addPrimaryServicePointNameToStaffSlipContext:: parameters entries: {}", entries);
+    if (primaryServicePoint == null) {
+      log.info("primaryServicePoint object is null");
+      return entries;
+    }
+
+    if (entries == null) {
+      log.info("entries JsonObject is null, primaryServicePointName: {}", primaryServicePoint.getName());
+      return new JsonObject();
+    }
+
+    log.debug("addPrimaryServicePointNameToStaffSlipContext:: primaryServicePointName: {}", primaryServicePoint.getName());
```

Comment:
it appears to be redundant

Reply:
This prints primaryServicePointName (this is an important field that we want to make sure is shown in logs)